feature-libs/asm/root/services/csagent-auth.service.ts

```diff
@@ -83,6 +83,43 @@ export class CsAgentAuthService {
     }
   }
 
+  /**
+   * Loads access token for a customer support agent when use code flow.
```

Comment:
```suggestion
   * Loads access token for a customer support agent when use code flow.
```
Loads access token for a customer support agent with authorization code flow.